CMakeLists.txt

```diff
@@ -3,9 +3,11 @@ project(module_a)
 
 set(CMAKE_C_STANDARD 99)
 
-add_library(module_a src/module_a.c)
-target_include_directories(module_a PUBLIC include)
-
+add_library(module_a SHARED src/module_a.c)
+target_include_directories(module_a PUBLIC
```

Comment:
This is a recommended way to define the include directories valid for both the build and the install layout

Reply:
oh no there are more lines here...

Reply:
Yep, multiline :)